helm-app-operator/pkg/helm/helm.go

```diff
@@ -506,3 +511,38 @@ func shortenUID(uid types.UID) (shortUID string) {
 	shortUID = strings.ToLower(base36.EncodeBytes(uidBytes))
 	return
 }
+
+func diff(a, b string) string {
+	dmp := diffmatchpatch.New()
+	wSrc, wDst, warray := dmp.DiffLinesToRunes(a, b)
+	diffs := dmp.DiffMainRunes(wSrc, wDst, false)
+	diffs = dmp.DiffCharsToLines(diffs, warray)
+	var buff bytes.Buffer
+	for _, diff := range diffs {
+		text := diff.Text
+		switch diff.Type {
+		case diffmatchpatch.DiffInsert:
+			_, _ = buff.WriteString("\x1b[32m")
```

Comment:
@AlexNPavel I recall we had some issues in seeing the colorized output while doing a diff for our unit tests. Do you remember if that was specific to what type of shell you use.

@joelanford Are you able to see the colorized output properly when you get the logs? Can you share a sample output?

Reply:
Most shells should support colors. Some shells may not, and this print out `[32m` in the output. This shouldn't really happen though, as almost all shells support color (or at least ignore the entire block defining the color, which is fine here since we are doing line based diffs)